scripts/docker-inner.sh

```diff
@@ -58,20 +57,14 @@ for version in $versions; do
     rm -rf $download_dir
     mkdir -p $download_dir
 
-    git clone https://github.com/drush-ops/drush.git $download_dir/$name
-    [ $? == 0 ] || (echo "Git repository could not be cloned!"; exit 1)
-    
-    cd $download_dir/$name
-    composer install
-    [ $? == 0 ] || (echo "Install failed!"; exit 1)
-    cd -
+    curl -L -f https://github.com/drush-ops/drush/releases/download/${version}/drush.phar --output $download_dir/$name.phar
+
+    [ $? == 0 ] || (echo "Download failed!"; exit 1)
```

Comment:
`(...)` forks a subshell, which is not what you want. The subshell may exit 1 but the outer script will continue (actually it won't continue because it has `set -e` but if that was removed then the behavior would be not what you intended.. You want a function with uses curly braces `{...}` here.